frontend/app/views/fragments/form/submit.scala.html

```diff
@@ -37,7 +39,14 @@
             <noscript>
                 <a href="http://www.enable-javascript.com/" target="_blank" class="action is-disabled">Enable javascript to proceed</a>.
             </noscript>
-            <button type="button" class="action js-submit-input u-hidden-non-js">
+            <button type="button" class="action js-submit-input u-hidden-non-js"
+                    @if(dataCategory != None && dataLabel != None) {
```

Comment:
If you see `if` statements and `Option`s together, you can usually guess there is a more scala-ish way of doing things! eg, use a for-comprension:

```
@for(
  label <- dataLabel
  category <- dataCategory
) {
  data-metric-category=@category
  data-metric-label=@dataLabel
}
```